### ISSUE TITLE
Support for keeping CSS in separate files

### ISSUE THREAD

**Issue by ythecombinator** [giuseppeg/styled-jsx-postcss#22]
Is [this feature](https://github.com/zeit/styled-jsx#keeping-css-in-separate-files) currently supported?

I have the following in my `styles.js`:

```js
const styles = `

@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro');
@import url(//fonts.googleapis.com/css?family=Fira+Sans:300);

html, body {
  margin: 0;
  padding: 0; }

...
  
`
export default styles
```

And when I try to import it in my component via

```js
import styles from './styles'
...
    <style jsx>
      { styles }
    </style>
```

I end up getting the following error:

```
Syntax Error: <style jsx>{`some css`}</style>), but got Identifier
```

My guess would be that it probably happens because [external StyleSheets support](https://github.com/zeit/styled-jsx/pull/148) came after this package's last update–which still sounds strange since you authored the PR on styled-jsx 😄 .

Any ideas on what's happening?

**Comment by giuseppeg** [comment]
yep this packages is using `0.5.7` so unfortunately external styles are not supported.
